web-ui/src/main/resources/catalog/locales/en-admin.json

```diff
@@ -1364,6 +1364,8 @@
     "ui-mod-recordview": "Record View",
     "ui-isSocialbarEnabled": "Show Social bar",
     "ui-isSocialbarEnabled-help": "Show the button bar with all the social buttons (facebook, twitter, etc.)",
+    "ui-isDefaultContactViewEnabled": "Show Default Contact View",
+    "ui-isDefaultContactViewEnabled-help": "Show the default contact full view instead of icon view",
```

Comment:
Similar comment. Maybe `Show the full contact details view instead of the icon view`

Reply:
yes, this makes sense instead of default. I have made the change. Please check